intelmq/lib/upgrades.py

```diff
@@ -610,6 +611,39 @@ def v300_defaults_file_removal(defaults, runtime, harmonization, dry_run):
     return messages if messages else changed, defaults, runtime, harmonization
 
 
+def v300_pipeline_file_removal(defaults, runtime, harmonization, dry_run):
+    """
+    Remove the pipeline.conf file
+    """
+    changed = None
+    messages = []
+    pipeline_file = Path(CONFIG_DIR) / "pipeline.conf"
+    if pipeline_file.exists():
+        if dry_run:
+            print('Would now remove file {pipeline_file!r}.')
```

Comment:
Can we move that downwards to the actual unlink command?

Reply:
Done.